doc/testing.rst

```diff
@@ -36,6 +36,44 @@ your request yourself:
                     resp = webapp.post('/some/nonexistent/url')
                     assert resp.status_code == requests.codes.not_found
 
+
+If you'd like to import a module from your project inside your tests, use:
+
+.. code-block:: python
+        def test_something(webapp):
+            from _cob import <module_name>
+            ...
+
+
+For example:
+
+.. code-block:: python
+        # file: models.py
+
+        # cob: type=models
+        from cob import db
+
+
+        class Person(db.Model):
+            id = db.Column(db.Integer, primary_key=True)
+            first_name = db.Column(db.String)
+            last_name = db.Column(db.String)
+
+            @property
+            def full_name(self):
+                return f"{self.first_name} {self.last_name}"
+
+
+.. code-block:: python
+
+        # file: tests/test_models.py
+
+        def test_person(webapp):
+            from _cob import models
```

Comment:
This is great, but looking forward it makes me wonder if we can do better and allow this import at the module-level. The only reason why this has to be inside of the test is because we don't load the project "early enough". I think we should research if this is fixable at `cob test` itself, making usage here more ergonomic

Reply:
I think I fixed it. I added a hook ("pytest_configure") that sets up the app. The hook is triggered before the test modules are imported.

Reply:
@vmalloc could you please review this?